### DISCUSSION TITLE
Multiple Computers

### DISCUSSION THREAD

**Discussion by Drallas** [lra/mackup#1770] [General]
Can this be used with Multiple Computers without corrupting dot files?

**Comment by joshmedeski** [comment]
Yes! That's of the major selling points to using Mackup. I even have it working across and Mac and Linux machines.

**Reply by Drallas** [reply]
Tnx don't want to mess-up my 'painfully' crafted dot file. But will run Mackup on my 2nd Mac.

**Reply by joshmedeski** [reply]
It's pretty easy to undo Mackup with `mackup restore`. So if you're not happy with how Mackup works you can go back to normal.
